Application/EdFi.Ods.Features/ChangeQueries/Providers/SnapshotOdsDatabaseConnectionStringProviderDecorator.cs

```diff
@@ -0,0 +1,65 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.Ods.Common.Configuration;
+using EdFi.Ods.Common.Context;
+using EdFi.Ods.Common.Database;
+using EdFi.Ods.Features.ChangeQueries.SnapshotContext;
+using log4net;
+
+namespace EdFi.Ods.Features.ChangeQueries.Providers
+{
+    public class SnapshotOdsDatabaseConnectionStringProviderDecorator : IOdsDatabaseConnectionStringProvider
+    {
+        private readonly IOdsDatabaseConnectionStringProvider _next;
+        private readonly IContextProvider<SnapshotUsage> _snapshotContextProvider;
+        private readonly IContextProvider<OdsInstanceConfiguration> _odsInstanceContextProvider;
+        private readonly ILog _logger = LogManager.GetLogger(typeof(OdsDatabaseConnectionStringProvider));
+
+        public SnapshotOdsDatabaseConnectionStringProviderDecorator(IOdsDatabaseConnectionStringProvider next,
```

Comment:
Bring the `IOdsDatabaseConnectionStringProvider` parameter to the next line, aligned with the other parameters.